charts/backstage/templates/app-config.yaml

```diff
@@ -76,7 +76,9 @@ stringData:
 
     auth:
       # see https://backstage.io/docs/auth/ to learn about auth providers
-      providers: {}
+      {{- with .Values.backstage.auth }}
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
```

Comment:
This was probably not meant to be included in this PR.

Reply:
Arrr I forgot that I pushed all the stuff in the main branch. I will close this PR and open a new PR

Reply:
I reverted back